mesonbuild/build.py

```diff
@@ -1160,12 +1161,21 @@ def __init__(self, name, subdir, subproject, is_cross, sources, objects, environ
         # The import library that GCC would generate (and prefer)
         self.gcc_import_filename = None
 
+        # check for modularity
+        self.is_modular = False
+        if 'modular' in kwargs and kwargs['modular']:
+            self.is_modular = True
+        if 'implib' in kwargs and kwargs['implib']:
+            self.is_modular = True
+        if self.is_modular and ('implib' in kwargs and kwargs['implib'] == False):
```

Comment:
__[Flake8]__

comparison to False should be 'if cond is False:' or 'if not cond:'


<!-- Comment by [SideCI](https://sideci.com) -->